src/tmr/tmr.c

```diff
@@ -446,6 +449,20 @@ void tmr_start_dbg(struct tmr *tmr, uint64_t delay, tmr_h *th, void *arg,
 }
 
 
+void tmr_start_dbg(struct tmr *tmr, uint64_t delay, tmr_h *th, void *arg,
+		   const char *file, int line)
+{
+	tmr_startcont_dbg (tmr, delay, true, th, arg, file, line);
+}
+
+
+void tmr_continue_dbg(struct tmr *tmr, uint64_t delay, tmr_h *th, void *arg,
+		   const char *file, int line)
+{
+	tmr_startcont_dbg (tmr, delay, false, th, arg, file, line);
```

Comment:
```suggestion
	tmr_startcont_dbg(tmr, delay, false, th, arg, file, line);
```

Reply:
Yes, those are my thumbs thinking for themselves.  Fixed in 8d6a3b384b0d97c52a1f5231bf47ed5cbf6e1261.